#! /bin/sh /usr/share/dpatch/dpatch-run
## 49-reduce-doxygen-doc.dpatch by Marek Brudka <mbrudka@aster.pl>
##
## DP: Enforce doxygen to generate smaller doxumentation

@DPATCH@
--- ACE_wrappers/etc/ace.doxygen-	2010-05-15 19:05:16.160407681 +0200
+++ ACE_wrappers/etc/ace.doxygen	2010-05-15 19:11:01.425651792 +0200
@@ -87,7 +87,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -159,11 +159,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -172,4 +176,4 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
--- ACE_wrappers/etc/ace_qos.doxygen-	2010-05-15 19:05:16.164407073 +0200
+++ ACE_wrappers/etc/ace_qos.doxygen	2010-05-15 19:10:54.532417597 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -147,11 +147,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -160,5 +164,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/etc/ace_rmcast.doxygen-	2010-05-15 19:05:16.168405627 +0200
+++ ACE_wrappers/etc/ace_rmcast.doxygen	2010-05-15 19:10:48.593645277 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -147,11 +147,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -160,5 +164,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/etc/ace_ssl.doxygen-	2010-05-15 19:05:16.172389793 +0200
+++ ACE_wrappers/etc/ace_ssl.doxygen	2010-05-15 19:10:42.316417037 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -147,11 +147,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -160,5 +164,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/etc/acexml.doxygen-	2010-05-15 19:05:16.176389953 +0200
+++ ACE_wrappers/etc/acexml.doxygen	2010-05-15 19:10:36.545642322 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -147,11 +147,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -160,5 +164,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/CIAO/etc/ciao_config_handlers.doxygen-	2010-05-15 19:05:16.176389953 +0200
+++ ACE_wrappers/TAO/CIAO/etc/ciao_config_handlers.doxygen	2010-05-15 19:10:30.217642894 +0200
@@ -76,7 +76,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -156,11 +156,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -169,5 +173,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/CIAO/etc/ciao_DAnCE.doxygen-	2010-05-15 19:05:16.180407713 +0200
+++ ACE_wrappers/TAO/CIAO/etc/ciao_DAnCE.doxygen	2010-05-15 19:10:22.337634604 +0200
@@ -76,7 +76,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -156,11 +156,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -169,4 +173,4 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
--- ACE_wrappers/TAO/CIAO/etc/ciao_dds4ccm.doxygen-	2010-05-15 19:05:16.184407245 +0200
+++ ACE_wrappers/TAO/CIAO/etc/ciao_dds4ccm.doxygen	2010-05-15 19:10:11.497643759 +0200
@@ -76,7 +76,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -156,11 +156,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -169,4 +173,4 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
--- ACE_wrappers/TAO/CIAO/etc/ciao.doxygen-	2010-05-15 19:05:16.188389595 +0200
+++ ACE_wrappers/TAO/CIAO/etc/ciao.doxygen	2010-05-15 19:10:04.305637668 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -157,11 +157,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -170,5 +174,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_anytypecode.doxygen-	2010-05-15 19:05:16.192390244 +0200
+++ ACE_wrappers/TAO/etc/tao_anytypecode.doxygen	2010-05-15 19:09:57.929643498 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_av.doxygen-	2010-05-15 19:05:16.196393966 +0200
+++ ACE_wrappers/TAO/etc/tao_av.doxygen	2010-05-15 19:09:52.088419478 +0200
@@ -80,7 +80,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -154,11 +154,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -167,5 +171,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_compression.doxygen-	2010-05-15 19:05:16.200391123 +0200
+++ ACE_wrappers/TAO/etc/tao_compression.doxygen	2010-05-15 19:09:46.072425406 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_cosevent.doxygen-	2010-05-15 19:05:16.204390933 +0200
+++ ACE_wrappers/TAO/etc/tao_cosevent.doxygen	2010-05-15 19:09:39.785639718 +0200
@@ -80,7 +80,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -155,11 +155,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -168,5 +172,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_cosnaming.doxygen-	2010-05-15 19:05:16.204390933 +0200
+++ ACE_wrappers/TAO/etc/tao_cosnaming.doxygen	2010-05-15 19:09:33.585634481 +0200
@@ -78,7 +78,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -151,11 +151,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -164,5 +168,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_cosnotification.doxygen-	2010-05-15 19:05:16.204390933 +0200
+++ ACE_wrappers/TAO/etc/tao_cosnotification.doxygen	2010-05-15 19:09:25.681626986 +0200
@@ -85,7 +85,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -161,11 +161,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -174,5 +178,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_costime.doxygen-	2010-05-15 19:05:16.208388020 +0200
+++ ACE_wrappers/TAO/etc/tao_costime.doxygen	2010-05-15 19:09:14.392425087 +0200
@@ -78,7 +78,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -152,11 +152,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -165,5 +169,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_costrader.doxygen-	2010-05-15 19:05:16.208388020 +0200
+++ ACE_wrappers/TAO/etc/tao_costrader.doxygen	2010-05-15 19:09:04.753631947 +0200
@@ -81,7 +81,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -155,11 +155,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -168,5 +172,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao.doxygen-	2010-05-15 19:05:16.208388020 +0200
+++ ACE_wrappers/TAO/etc/tao.doxygen	2010-05-15 19:08:58.233629350 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -149,11 +149,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -162,5 +166,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_dynamicany.doxygen-	2010-05-15 19:05:16.212388390 +0200
+++ ACE_wrappers/TAO/etc/tao_dynamicany.doxygen	2010-05-15 19:08:52.241629601 +0200
@@ -78,7 +78,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_dynamicinterface.doxygen-	2010-05-15 19:05:16.212388390 +0200
+++ ACE_wrappers/TAO/etc/tao_dynamicinterface.doxygen	2010-05-15 19:08:43.577627809 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -149,11 +149,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -162,5 +166,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_esf.doxygen-	2010-05-15 19:05:16.216387851 +0200
+++ ACE_wrappers/TAO/etc/tao_esf.doxygen	2010-05-15 19:08:37.865621183 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -151,11 +151,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -164,5 +168,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_ifr.doxygen-	2010-05-15 19:05:16.216387851 +0200
+++ ACE_wrappers/TAO/etc/tao_ifr.doxygen	2010-05-15 19:08:31.969632450 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -153,11 +153,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -166,5 +170,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_implrepo.doxygen-	2010-05-15 19:05:16.216387851 +0200
+++ ACE_wrappers/TAO/etc/tao_implrepo.doxygen	2010-05-15 19:08:26.428418559 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -152,11 +152,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -165,5 +169,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_iormanip.doxygen-	2010-05-15 19:05:16.220389268 +0200
+++ ACE_wrappers/TAO/etc/tao_iormanip.doxygen	2010-05-15 19:08:20.492417265 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_iortable.doxygen-	2010-05-15 19:05:16.220389268 +0200
+++ ACE_wrappers/TAO/etc/tao_iortable.doxygen	2010-05-15 19:08:14.624412942 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_pi.doxygen-	2010-05-15 19:05:16.220389268 +0200
+++ ACE_wrappers/TAO/etc/tao_pi.doxygen	2010-05-15 19:08:06.865623350 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_pi_server.doxygen-	2010-05-15 19:05:16.224388241 +0200
+++ ACE_wrappers/TAO/etc/tao_pi_server.doxygen	2010-05-15 19:08:00.809622380 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_portablegroup.doxygen-	2010-05-15 19:05:16.224388241 +0200
+++ ACE_wrappers/TAO/etc/tao_portablegroup.doxygen	2010-05-15 19:07:54.633621934 +0200
@@ -80,7 +80,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -154,11 +154,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -167,5 +171,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_portableserver.doxygen-	2010-05-15 19:05:16.224388241 +0200
+++ ACE_wrappers/TAO/etc/tao_portableserver.doxygen	2010-05-15 19:07:48.112419248 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_pss.doxygen-	2010-05-15 19:05:16.228387912 +0200
+++ ACE_wrappers/TAO/etc/tao_pss.doxygen	2010-05-15 19:07:40.400425824 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -151,11 +151,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -164,5 +168,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_rtcorba.doxygen-	2010-05-15 19:05:16.228387912 +0200
+++ ACE_wrappers/TAO/etc/tao_rtcorba.doxygen	2010-05-15 19:07:33.193620062 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_rtevent.doxygen-	2010-05-15 19:05:16.228387912 +0200
+++ ACE_wrappers/TAO/etc/tao_rtevent.doxygen	2010-05-15 19:07:26.449618668 +0200
@@ -86,7 +86,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -161,11 +161,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -174,5 +178,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_rtportableserver.doxygen-	2010-05-15 19:05:16.232389049 +0200
+++ ACE_wrappers/TAO/etc/tao_rtportableserver.doxygen	2010-05-15 19:07:19.353617003 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -150,11 +150,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -163,5 +167,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_security.doxygen-	2010-05-15 19:05:16.232389049 +0200
+++ ACE_wrappers/TAO/etc/tao_security.doxygen	2010-05-15 19:07:12.425616877 +0200
@@ -82,7 +82,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -156,11 +156,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -169,5 +173,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_smartproxies.doxygen-	2010-05-15 19:05:16.236387812 +0200
+++ ACE_wrappers/TAO/etc/tao_smartproxies.doxygen	2010-05-15 19:07:05.697614982 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -149,11 +149,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -162,5 +166,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_ssliop.doxygen-	2010-05-15 19:05:16.236387812 +0200
+++ ACE_wrappers/TAO/etc/tao_ssliop.doxygen	2010-05-15 19:06:58.385608300 +0200
@@ -82,7 +82,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -157,11 +157,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -170,5 +174,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_strategies.doxygen-	2010-05-15 19:05:16.236387812 +0200
+++ ACE_wrappers/TAO/etc/tao_strategies.doxygen	2010-05-15 19:06:50.945619688 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -149,11 +149,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -162,5 +166,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_transportcurrent.doxygen-	2010-05-15 19:05:16.240388042 +0200
+++ ACE_wrappers/TAO/etc/tao_transportcurrent.doxygen	2010-05-15 19:06:42.913618695 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -151,11 +151,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -164,5 +168,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
--- ACE_wrappers/TAO/etc/tao_ziop.doxygen-	2010-05-15 19:05:16.240388042 +0200
+++ ACE_wrappers/TAO/etc/tao_ziop.doxygen	2010-05-15 19:06:31.465239644 +0200
@@ -77,7 +77,7 @@
 HTML_FOOTER           =
 HTML_STYLESHEET       =
 HTML_ALIGN_MEMBERS    = YES
-GENERATE_HTMLHELP     = YES
+GENERATE_HTMLHELP     = NO
 
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
@@ -151,11 +151,15 @@
 
 HAVE_DOT              = YES
 CLASS_GRAPH           = YES
-COLLABORATION_GRAPH   = YES
-INCLUDE_GRAPH         = YES
-INCLUDED_BY_GRAPH     = YES
+COLLABORATION_GRAPH   = NO
+INCLUDE_GRAPH         = NO
+INCLUDED_BY_GRAPH     = NO
 GRAPHICAL_HIERARCHY   = YES
 DOT_MULTI_TARGETS     = YES
+DOT_GRAPH_MAX_NODES   = 15
+MAX_DOT_GRAPH_DEPTH   = 2
+DOT_IMAGE_FORMAT      = png
+
 # The tag DOT_PATH can be used to specify the path where the dot tool can be
 # found. If left blank, it is assumed the dot tool can be found on the path.
 DOT_PATH              =
@@ -164,5 +168,5 @@
 # Configuration::additions related to the search engine
 #---------------------------------------------------------------------------
 
-SEARCHENGINE          = YES
+SEARCHENGINE          = NO
 
